pages/server/create.mdx

```diff
@@ -1,18 +1,18 @@
 # Server creation
 
-We have over **24** different server types you can choose from to start your hosting experience!
+We have over **49** different server types you can choose from to start your hosting experience!
 
 <Tip>
   For the purpose of this page we will just be showing the create server list,
   you can look at the other guides to see how to create and work with them
 </Tip>
 
 (For non-donators)
-By running `DBH!server create list` you can see the list of all the servers you can create!
```

Comment:
The command is actually `DBH!server list` no longer `DBH!server create` as the way it was wrote is that the list command shows everything that the images show.

Reply:
ah ok thank you

Reply:
wait but server list shows the person's servers, when the images show the server list?

Reply:
Hmm guess that whole command got changed, guess it is just create